.github/workflows/ci.yml

```diff
@@ -253,12 +256,7 @@ jobs:
         run: npm run test -- --test-suite ${{ matrix.spec.suite }}
       - name: Test bundling and installation
         if: ${{ matrix.spec.machine == 'ubuntu-latest' }}
-        run: |
-          # Test prod build instead of dev.
-          npm run clean
-          npm run build
-          # Note: this modifies package.json to test puppeteer-core, so we test this last.
-          npm run test:install
+        run: npm run test:install
```

Comment:
Let's keep the prod build here as I think the test build might drift away and produce broken builds.

Reply:
Let's optimize this later. Not sure if it makes sense to optimize this now since it will slow down CI without any reason.

Reply:
Well, let's refactor this later then and keep the current code?